cico_run_bdd_tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
yes at least for the time where the BDD tests don't run stable when much of the code changes or not all developers know how the system works.